.gitignore

```diff
@@ -35,3 +35,9 @@ src/applications/bmqbrkr/etc/etc
 
 # 'sim_cpp11_features.pl' backups
 *.bak
+.bash_history
+.bde_tools/
+docker/sanitizers/
+lib64/
+src/groups/mqb/mqbc/mqbc_clusterutil.t.cpp
+src/groups/mqb/mqbc/mqbc_clusterutil.t.cpp
```

Comment:
Among these newly added items:

1. Some are not expected in a normal workflow:
```
.bash_history
.bde_tools/
lib64/
```

2. Some are legit project-related files or folders and there is no reason to ignore them:
```
docker/sanitizers/
src/groups/mqb/mqbc/mqbc_clusterutil.t.cpp
src/groups/mqb/mqbc/mqbc_clusterutil.t.cpp
```

Let's revert changes in this file